org.eclipse.jdt.core/search/org/eclipse/jdt/internal/core/search/indexing/AddJarFileToIndex.java

```diff
@@ -318,6 +317,6 @@ public String toString() {
 	}
 
 	protected boolean hasPreBuiltIndex() {
-		return !this.forceIndexUpdate && (this.indexFileURL != null && this.indexFileURL.exists());
+		return !this.forceIndexUpdate && ((this.indexFileURL != null) && this.indexFileURL.exists());
```

Comment:
See above: is this an auto-formatting result? Maybe remove at least the first added pair of brackets?